src/config/dynamic-theme-fixes.config

```diff
@@ -14648,6 +14648,36 @@ a.logo[title="United Nations"]
 
 ================================
 
+uokik.gov.pl
+
+INVERT
+.a-press-office
+.abgLong
+h1[id="logo"]
+p a
+p.nav img
+span[class="number"]
+ul[class="tabs"]
+
+CSS
+#subpage,
+.sWrapper1,
+.sWrapper2,
+.tabs,
+body#page,
+div#right,
+div#top,
+div.fWrapper,
+div[class="b1"],
+h2#here {
+    background-image: none !important;
+}
+#footer .active {
+    color: #004fb5 !important;
```

Comment:
Could this be made in the form of `${light variant of the color}`?

Reply:
Will do later. I am AFK. Regards